notebooks/growth_model_forecast.ipynb

```diff
@@ -0,0 +1,1186 @@
+{
```

Comment:
<p>remove: <code>We will be fetching data to obtain the monthly revenue for all accounts.</code></p><p> </p><p>maybe also add an additional line to describe the data: </p><p><code>The dataset consists of user/customer IDs, month and the revenue for each month.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>can you remove this line: <code>We will be fetching data to obtain the monthly revenue for all accounts.</code>?</p>